libs/portals/my-pages/social-insurance-maintenance/src/components/PaymentGroupTable/PaymentGroupTable.tsx

```
@@ -1,99 +1,130 @@
+                background: 'blue100',
+              }}
+            >
+              <Text textAlign="right" variant="medium" fontWeight="medium">
+                {formatMessage(m.year)}
+              </Text>
+            </T.HeadData>
+          </T.Row>
+        </T.Head>
+        <T.Body>
+          {paymentGroups?.map((paymentGroup) => {
+            return <PaymentGroupTableRow paymentGroup={paymentGroup} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)